packages/web/src/common/store/pages/signon/sagas.ts

```diff
@@ -565,340 +574,285 @@ function* sendRecoveryEmail({
-            shouldToast: rateLimited
-          }
-          if (rateLimited) {
-            params.message = 'Please try again later'
-            yield* put(
-              make(Name.CREATE_ACCOUNT_RATE_LIMIT, {
-                handle,
-                email,
-                location
-              })
-            )
-            reportToSentry({
-              error,
-              level: ErrorLevel.Warning,
-              name: 'Sign Up: User rate limited',
-              additionalInfo: {
-                handle,
-                email,
-                location,
-                formFields: createUserMetadata,
-                hasWallet: alreadyExisted
-              }
-            })
-          } else if (blocked) {
-            params.message = 'User was blocked'
-            params.uiErrorCode = UiErrorCode.RELAY_BLOCKED
             yield* put(
-              make(Name.CREATE_ACCOUNT_BLOCKED, {
-                handle,
+              identify(handle, {
+                name,
                 email,
-                location
+                userId
               })
             )
-            reportToSentry({
-              error,
-              level: ErrorLevel.Warning,
-              name: 'Sign Up: User was blocked',
-              additionalInfo: {
-                handle,
-                email,
-                location,
-                formFields: createUserMetadata,
-                hasWallet: alreadyExisted
-              }
-            })
-          } else {
-            reportToSentry({
-              error,
-              level: ErrorLevel.Error,
-              name: 'Sign Up: Other Error',
-              additionalInfo: {
-                handle,
-                email,
-                location,
-                formFields: createUserMetadata,
-                hasWallet: alreadyExisted
-              }
-            })
-          }
-          yield* put(signOnActions.signUpFailed(params))
-        }
-      },
-      function* () {
-        yield* put(signOnActions.sendWelcomeEmail(name))
-        yield* call(fetchAccountAsync, { isSignUp: true })
-        yield* put(signOnActions.followArtists())
-        yield* put(make(Name.CREATE_ACCOUNT_COMPLETE_CREATING, { handle }))
-        yield* put(signOnActions.signUpSucceeded())
-      },
-      function* ({ timeout, error, message }) {
-        if (timeout) {
-          console.debug('Timed out trying to register')
-          yield* put(signOnActions.signUpTimeout())
-        }
-        if (error) {
-          const reportToSentry = yield* getContext('reportToSentry')
-          reportToSentry({
-            error,
-            name: 'Sign Up: Error in signUp saga',
-            additionalInfo: { message, timeout }
-          })
-        }
-        if (message) {
-          console.debug(message)
-        }
-      },
-      () => {},
-      SIGN_UP_TIMEOUT_MILLIS
-    )
-  )
-}
-
-/**
- * Repairs broken signups from #flare-206
- */
-function* repairSignUp() {
```

Comment:
is this unnecessary now?

Reply:
yea - discussion is in client-money slack but general thinking was its ok to remove